.rubocop.yml

```diff
@@ -1,3 +1,7 @@
+require:
```

Comment:
this is copied from mco repo or where? Ideally we sync the style between that repo and these (I think I did that when i got them from Puppet Labs)

Reply:
Yes it was copied from choria-io/mcorpc-ruby-support:

```diff
--- /home/romain/Projects/choria-io/mcorpc-ruby-support/.rubocop.yml	2022-02-10 11:18:38.005071305 -1000
+++ .rubocop.yml	2023-04-04 13:30:07.403371049 -1000
@@ -1,8 +1,10 @@
+require:
+  - rubocop-performance
+  - rubocop-rake
+
 AllCops:
-  TargetRubyVersion: "2.4"
   DisplayCopNames: true
   DisplayStyleGuide: true
-  SuggestExtensions: false
   Exclude:
     - "module/**/*"
     - "vendor/**/*"
@@ -107,7 +109,7 @@
   Enabled: false
 
 Layout/LineLength:
-  Max: 180
+  Max: 200
 
 Metrics/PerceivedComplexity:
   Enabled: false
```